packages/teams-js/src/public/runtime.ts

```diff
@@ -192,6 +194,13 @@ export const versionConstants: Record<string, Array<ICapabilityReqs>> = {
       hostClientTypes: [HostClientType.android, HostClientType.desktop, HostClientType.ios],
     },
   ],
+
+  '2.2.0': [
```

Comment:
Can you say more about this very part of your change? @aeunms would know best, but I think it would mean legacy Teams clients understand your new API, which I don't think is the case?

Reply:
I have fixed this in the latest commit.